### ISSUE TITLE
gastrocnemius contracture

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7334]
**Preferred term label:**
Gastrocnemius contracture

**Synonyms**
Gastroc contracture

**Definition (free text, please give PubMed ID)**
Limitation of simultaneous ankle dorsiflexion and knee extension during mid-stance of gait (PMID: 22819670)

**Parent term (use hpo.jax.org/app)**
Flexion contracture HP:0001371

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
We are currently working with the AMC group to revise the entire contracture terminology. Let's wait for that workshop to add this term.

**Comment by pnrobinson** [comment]
@shahrzadnema
Can we discuss where this fits into the new terms?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000366
- label: Gastrocnemius contracture
- parents: [Lower-limb joint contracture (HP:0005750)](https://hpo.jax.org/app/browse/term/HP:0005750)
- pmid: PMID:22819670
- def: Limitation of simultaneous ankle dorsiflexion and knee extension during mid-stance of gait.
- comment: 
- orcid: orcid:0000-0002-6670-9157
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7334
- synonyms:
